ui/panels-plugin/src/plugins/line-chart/tooltip/tooltip-model.ts

```diff
@@ -13,13 +13,14 @@
 
 import { FocusedSeriesArray } from '../utils/focused-series';
 
-export const TOOLTIP_MIN_WIDTH = 360;
+export const TOOLTIP_MAX_WIDTH = 700;
+export const TOOLTIP_MAX_HEIGHT = 600;
 
```

Comment:
I updated the max width and max height considerably, but I feel like this is a better default strategy than allowing user resizing.

Reply:
Yeah for sure, the user resizing thing was a dead-end that I should have just removed at the time :smile: